<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PointsDialog</class>
 <widget class="QDialog" name="PointsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="pointsTable">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>30</y>
     <width>388</width>
     <height>172</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QTableWidget {
        background-color: #ffffff;
        border: 1px solid #e0e0e0;
        border-radius: 8px;
        padding: 5px;
        gridline-color: #f0f0f0;
    }
    /* 表头：浅灰色背景+加粗文字 */
    QHeaderView::section {
        background-color: #f8f9fa;
        font-weight: bold;
        color: #333333;
        padding: 8px;
        border: none;
        border-bottom: 1px solid #e0e0e0;
    }
    /* 表格行：隔行变色 */
    QTableWidget::item:even {
        background-color: #fcfcfc;
    }
    QTableWidget::item:odd {
        background-color: #f9f9f9;
    }
    /* 鼠标 hover 行：高亮 */
    QTableWidget::item:hover {
        background-color: #e6f7ff;
    }
    /* 表格文字：统一字体 */
    QTableWidget::item {
        padding: 6px;
        color: #333333;
    }</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
   </property>
   <property name="columnCount">
    <number>4</number>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>积分类型</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>积分数量</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>获取时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>备注</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="totalPointsLabel">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>6</y>
     <width>388</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>150</horstretch>
     <verstretch>20</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>20</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:16px;
color:#FF6700;
font-weight:bold;</string>
   </property>
   <property name="text">
    <string>当前总积分：0</string>
   </property>
  </widget>
  <widget class="QLabel" name="ruleLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>321</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#666;
line-height:1.5;
padding: 10px;
        border-radius: 6px;
        border: 1px solid #f0f0f0;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;积分规则：1.注册赠送100积分       2.每消费1元累积1积分      3.积分可兑换优惠券/抵扣费用；&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>60</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#aaffff；
color:white;
border:none;
border-radius:4px;
padding:6px 12px;</string>
   </property>
   <property name="text">
    <string>刷新积分</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>